config/environments/profile.rb

```diff
@@ -0,0 +1,41 @@
+# an environment configured for performance profiling.
```

Comment:
There were a few things I wanted from dev. I can copy prod instead and put exceptions clearly labeled at the top.